mod/settings.lua

```diff
@@ -60,6 +60,17 @@ data:extend({
         localised_description = "Compression quality for jpg files.",
         order = "201",
     },
+    {
+        type = "int-setting",
+        name = "minjpgquality",
+        setting_type = "runtime-per-user",
+        default_value = 75,
+        minimum_value = 0,
+        maximum_value = 100,
+        localised_name = "Minimum JPG quality",
+        localised_description = "Compression quality for jpg files when no player entities. Set to 0 to skip that tile.",
```

Comment:
nit: "when no player entities. Set to 0 to skip that tile." -> "when no player entities are present. Set to 0 to skip the tile entirely."